tileview/src/main/java/com/qozix/tileview/markers/MarkerLayout.java

```diff
@@ -16,6 +16,10 @@
 
   private MarkerTapListener mMarkerTapListener;
 
+  public static final int POS_RELATIVE = 0;               // Position is relative
```

Comment:
the names of these flags aren't immediately clear to me.  maybe something like `ANCHOR_BEHAVIOR_RELATIVE` and `ANCHOR_BEHAVIOR_ABSOLUTE`?

Reply:
I'll rename these too.